app/views/layouts/cms.html.erb

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Admin CMS</title>
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
+    <%= stylesheet_link_tag    'application', 'https://fonts.googleapis.com/css?family=Muli' %>
+    <%= stylesheet_link_tag    'application', 'https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css' %>
+    <%= stylesheet_link_tag    'application', '/assets/css/app.css?a=1499756908' %>
+    <%= stylesheet_link_tag    'application', '/nsw-state-library-amplify/assets/css/styles.css?a=1499756908' %>
+    <%= stylesheet_link_tag    'application', '/assets/css/app.css?a=1499756908' %>
+
+    <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+
+    <%= csrf_meta_tags %>
+  </head>
+  <body>
+    <div id="app" class="container">
+      <div id="header" class="header">
+        <nav id="primary-navigation" class="navigation primare inverse" role="navigation">
+          <div>
+            <div id="header-title" class="title-container"></div>
+            <div id="header-crumbs" class="crumbs-container"></div>
+            <div id="account-container" class="account-container"></div>
+            <div id="primary-menu-container" class="menu-container"></div>
+          </div>
+        </nav>
+        <nav id="secondary-navigation" class="navigation secondary"></nav>
+        <div id="primary-alert" class="alert" role="alert" aria-atomic="true"></div>
+      </div>
+
+      <div id="main" class="body">
+        <table>
```

Comment:
Do we need to use `<table>` for layout? That's pretty discouraged these days. Could we achieve the same effect with `<div>`s?

```
<div class='body'>
  <div class='header'>
    <%= yield :cms_heading %>
  </div>

  <div class='content'>
    <%= yield %>
  </div>
</div>
```